Celeste.Mod.mm/Mod/Everest/Everest.Loader.cs

```diff
@@ -32,11 +32,11 @@ public static class Loader {
             /// The path to the Everest /Mods/blacklist.txt file.
             /// </summary>
             public static string PathBlacklist { get; internal set; }
-            internal static List<string> _Blacklist = new List<string>();
+            internal static HashSet<string> _Blacklist = new HashSet<string>();
             /// <summary>
             /// The currently loaded mod blacklist.
             /// </summary>
-            public static ReadOnlyCollection<string> Blacklist => _Blacklist?.AsReadOnly();
+            public static IReadOnlyCollection<string> Blacklist => (IReadOnlyCollection<string>) _Blacklist ;
```

Comment:
instead of a simple cast, I would use `ToImmutableHashSet` to make it actually read-only :thinking: 

also the type change might be a breaking change, even though I don't think any mods would use `Everest.Loader.Blacklist` (unless there's a cursed mod out there that does stuff based on the blacklist :maddyS:)